hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/DatanodeMetadata.java

```diff
@@ -67,6 +67,9 @@
   @XmlElement(name = "buildDate")
   private String buildDate;
 
+  @XmlElement(name = "layoutVersion")
```

Comment:
General question: Why is this POJO class replicated inside recon vs having just one version in hdds-common or somewhere higher.

Reply:
You are correct, this was added as a Recon specific entity that may be formed by picking metadata pieces from multiple internal representations, and to support JAXB (I believe). We can revisit it and see if we can just reuse internal entities.

Reply:
Make sense for another time but I think having all pojos in one place makes code so much more readable.